tests/forms_tests/tests/test_formsets.py

```diff
@@ -1197,6 +1197,61 @@ def test_html_safe(self):
         self.assertTrue(hasattr(formset, '__html__'))
         self.assertEqual(str(formset), formset.__html__())
 
+    def test_can_delete_extra_formset_forms(self):
+        class GenericForm(Form):
+            foo = CharField()
+
+        GenericFormFormset = formset_factory(
+            form=GenericForm,
+            can_delete=True,
+            extra=2)
+        formset = GenericFormFormset()
+        self.assertEqual(len(formset), 2)
+        self.assertIn('DELETE', formset.forms[0].fields)
+        self.assertIn('DELETE', formset.forms[1].fields)
+
+    def test_delete_fields_are_not_added_for_extra_formset_forms(self):
```

Comment:
I left as the original name, but could be `test_can_not_delete_extra_formset_forms` ?

Reply:
I renamed the test and removed the docstring.